src/libsyntax_ext/deriving/clone.rs

```diff
@@ -46,8 +48,8 @@ pub fn expand_deriving_clone(cx: &mut ExtCtxt,
             match annitem.node {
                 ItemKind::Struct(_, Generics { ref ty_params, .. }) |
                 ItemKind::Enum(_, Generics { ref ty_params, .. })
-                    if ty_params.is_empty()
-                        && attr::contains_name(&annitem.attrs, "derive_Copy") => {
+                    if ty_params.is_empty() &&
+                    attr::contains_name(&annitem.attrs, "derive_Copy") => {
```

Comment:
It's a bug, I filed https://github.com/rust-lang-nursery/rustfmt/issues/1099 for this